src/main/java/org/minions/devfund/benjamin/Movie.java

```diff
@@ -0,0 +1,73 @@
+package org.minions.devfund.benjamin;
+
+import java.util.ArrayList;
+
+/**
+ * Movie class.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     *
+     * Constructor.
+     */
+    public Movie() {
+       actors = new ArrayList<Actor>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)